speech/speech_synthesizer.go

```diff
@@ -110,6 +110,25 @@ func (synthesizer SpeechSynthesizer) SpeakTextAsync(text string) chan SpeechSynt
 	return outcome
 }
 
+// Execute the speech synthesis on SSML, asynchronously.
+func (synthesizer SpeechSynthesizer) SpeakSsmlAsync(ssml string) chan SpeechSynthesisOutcome {
```

Comment:
The name of the member needs to be the first word in the doc comment. Also, since SSML is an acronym, I would recommend using SpeakSSMLAsync.

ps. Have you tried running the linter?

Reply:
for the name, I agree with you to use `SpeakSSMLAsync`, But as other bindings are using `SpeakSsmlAsync`, I think we'd better to keep the alignment.

I didn't run the linter... I run it now and fix several issues. I also add the lint check to the workflow